src/bionics.cpp

```diff
@@ -1493,21 +1494,22 @@ void Character::burn_fuel( bionic &bio, const auto_toggle_bionic_result &result
 
     map &here = get_map();
     weather_manager &weather = get_weather();
+    // TODO: Rewrite fuels to use item volume.
+    // The energy is in energy/L but we just divide by 1000 and treat it as energy/unit
```

Comment:
isn't a unit 250 mL?

Reply:
Unit here means one unit of item (one charge of battery, one charge of gasoline, etc.).

Reply:
Yes but to convert volume into unit you need to know the ratio between them, no?

Reply:
Materials do not have volumes. That is the problem.

Reply:
Ah so it's just an arbitrary convertion untill there's a better fix?

Reply:
Yes.